srcpkgs/hydrus/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'hydrus'
+pkgname=hydrus
+version=502
+revision=1
+archs="x86_64"
+makedepends="python3"
+depends="python3 libopencv-python3 python3-BeautifulSoup4 python3-yaml python3-lz4
+ python3-numpy python3-Twisted python3-Pillow python3-pysocks python3-psutil
+ python3-send2trash python3-html5lib python3-requests python3-QtPy noto-fonts-emoji
+ python3-mpv python3-service_identity python3-PyQt5 fmt python3-pyside2 ffmpeg
+ miniupnpc desktop-file-utils python3-cloudscraper python3-openssl"
+short_desc="Danbooru-like image tagging and searching system for the desktop"
+maintainer="fanyx <fanyx@pm.me>"
+license="WTFPL"
+homepage="https://hydrusnetwork.github.io/hydrus/"
+distfiles="https://github.com/hydrusnetwork/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=684e643f781fa7682a670a75ee09fbc12531d27904be99d9b8e642f50c277d88
+
+pre_build() {
+	rm -f "hydrus/Test"*.py
+	rm -rf "static/testing"
+}
+
+do_build() {
+	echo 'Compiling .py files...'
+	python3 -OO -m compileall -fq .
+}
+
+do_install() {
+	vmkdir usr/lib/hydrus
+	cp -r hydrus static client.pyw server.py ${DESTDIR}/usr/lib/hydrus/
+
+	vbin ${FILESDIR}/hydrus-client
+	vbin ${FILESDIR}/hydrus-server
+
+	vmkdir usr/share/applications
+	vcopy ${FILESDIR}/hydrus.desktop usr/share/applications/
+
+	vsed -i hydrus/core/HydrusConstants.py -e 's|^LICENSE_PATH.*|LICENSE_PATH = "/usr/share/licenses/hydrus/license.txt"|'
```

Comment:
you are doing this after you installed the files, so this doesn't do anything

Reply:
switched order to sed at beginning of do_install